processing/data_collection/gazette/spiders/rn_macaiba.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+import dateparser
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMacaiba(BaseGazetteSpider):
+
+    TERRITORY_ID = "2407104"
+    name = "rn_macaiba"
+    allowed_domains = ["macaiba.rn.gov.br"]
+
+    def __init__(self, start_date=None, end_date=None):
+        self.start_date = dt.date.today().replace(day=1)
```

Comment:
Please, consider the `start_date` the first date with gazettes available. This is the default in other spiders.

Reply:
Done.